openspec/changes/archive/2026-02-17-docs-link-card-integration/proposal.md

```
@@ -0,0 +1,114 @@
+
+### 变更前后对比
+
+**变更前 - 传统列表链接**（以 installation/desktop.mdx 为例）```
+## 后续步骤
+
+完成安装后，您可以：
+
+- [创建第一个项目](/quick-start/create-first-project) - 初始化您的 Hagicode 项目
+- [创建普通会话](/quick-start/conversation-session) - 开始与 AI 交互
+- [创建提案会话](/quick-start/proposal-session) - 了解提案驱动开发
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)